libs/arrow-meta/src/main/kotlin/arrow/meta/internal/registry/InternalRegistry.kt

```diff
@@ -241,13 +245,11 @@ interface InternalRegistry : ConfigSyntax {
     )
   }
 
-  fun registerSyntheticScopeProvider(
-    project: Project,
+  fun CompilerPluginRegistrar.ExtensionStorage.registerSyntheticScopeProvider(
```

Comment:
This is because of the change in `CompilerPluginRegistrar`. Now `registerExtension` doesn't live in each component, but in `CompilerPluginRegistrar.ExtensionStorage`. So the code looks almost the same, but under the hood the registration process has changed.